src/States/atDanceGameState.js

```diff
@@ -82,10 +122,55 @@ function AtDanceGame() {
 
 	}
 
+	function touchItem1() {
```

Comment:
:space_invader: New line spacing at the start and end of `touchItem1()`, `touchItem2()`, and `touchItem3()` aren't needed